### PR TITLE
Update README-sycl.md

### PR DESCRIPTION
**Pull request by mehdi-goli**
Fixing CUDA version

### CODE REVIEW

**Review by AD2605 — APPROVED** [review]

**Review by aacostadiaz — APPROVED** [review]

File: README-sycl.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ ninja [EXAMPLE_NAME]_sycl
 You can run it like this from your build directory
 
 ```
+LD_LIBRARY_PATH=/path/to/sycl/install/lib
```
Comment (rolandschulz) [inline README-sycl.md:55]: if you split it over two lines you need export, don't you?
```suggestion
export LD_LIBRARY_PATH=/path/to/sycl/install/lib 
```

Note that you previously had it on the same line in
https://github.com/codeplaysoftware/cutlass-fork/pull/18/files
but then 
https://github.com/codeplaysoftware/cutlass-fork/pull/20 removed it before it was added back in here. 

(which made reviewing https://github.com/codeplaysoftware/cutlass-fork/pull/26 really confusing)

File: README-sycl.md [hunk 2]
```diff
@@ -62,7 +63,8 @@ You can run it like this from your build directory
  ```
  You can run it like this from your build directory
  ```
-  NVIDIA_TF32_OVERRIDE=1 ./examples/14_ampere_tf32_tensorop_gemm/14_ampere_tf32_tensorop_gemm_cute
+  NVIDIA_TF32_OVERRIDE=1 LD_LIBRARY_PATH=/path/to/sycl/install/lib
```
Comment (rolandschulz) [inline README-sycl.md:66]: same here

File: README-sycl.md (hunk 1 above)
Comment (mehdi-goli) [reply README-sycl.md:55]: Thanks @rolandschulz  for catching this. I did not realised that it has been broken over the line. Fixed it now in the new PR 
https://github.com/codeplaysoftware/cutlass-fork/pull/31

Comment (aacostadiaz) [reply README-sycl.md:55]: Yes, it should use 'export' if on a different line. Good catch, I'll create a PR for that.

**Pull request merged**